maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -59806,12 +59806,20 @@
 /turf/simulated/floor/plating,
 /area/eris/engineering/engine_room)
 "cKc" = (
-/obj/item/weapon/storage/briefcase/inflatable{
-	pixel_x = 3;
-	pixel_y = 3
+/obj/item/weapon/rig/techno,
+/obj/item/weapon/rig/techno,
+/obj/item/weapon/rig/techno,
```

Comment:
3 rigs is one too many. I think two is better. You'll never have more than 2 or 3 technomancers and rigs are stronger than the more available voidsuits

Reply:
> rigs are stronger than the more available voidsuits

they aren't.

Reply:
Firefox said 3 copies at eva storage sounds optimal.

Reply:
Ok